src/sass/components/_EditsNav.scss

```diff
@@ -28,6 +25,9 @@
       a {
         color: $color-primary;
         font-weight: 700;
+        margin: 0;
```

Comment:
I think this all gets cleaner if we ...

1. remove these lines
1. update the [`<hr>` ](https://github.com/cfpb/hmda-platform-ui/blob/64a37b8310fc99b1894ea5d6a48f3c3843b3392d/src/sass/components/_EditsNav.scss#L123) to set `top` to `9px`
1. add `.EditsNav ul.usa-nav-primary li { padding: 0 }` as a reset to USWDS (within this file)
1. remove [this line](https://github.com/cfpb/hmda-platform-ui/blob/64a37b8310fc99b1894ea5d6a48f3c3843b3392d/src/sass/components/_EditsNav.scss#L3)
1. remove [this line](https://github.com/cfpb/hmda-platform-ui/blob/64a37b8310fc99b1894ea5d6a48f3c3843b3392d/src/sass/components/_EditsNav.scss#L25)
1. set [this line](https://github.com/cfpb/hmda-platform-ui/blob/64a37b8310fc99b1894ea5d6a48f3c3843b3392d/src/sass/components/_EditsNav.scss#L50) to `margin: 0 auto`

This would remove the negative margins and keep the focus as it should be.

The spacing feels pretty good, improves the spacing between the nav and its border.

If you think the spacing between the user heading and the nav could shrink some, I wouldn't mind setting [the padding here](https://github.com/cfpb/hmda-platform-ui/blob/64a37b8310fc99b1894ea5d6a48f3c3843b3392d/src/sass/components/_UserHeading.scss#L2) to `1em 3rem 0`.